src/System.Text.Primitives/System/Text/Formatters/Utf16.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Text.Formatters
+{
+    public static partial class Utf16
+    {
+        #region Common constants
+
+        private const char Colon = ':';
```

Comment:
For this change, because I used partial classes, I moved anything that was "common" and used in multiple places into here... plus ones that I foresee as probably being needed in more than 1 type format.